src/BRFW.CPP

```diff
@@ -9,7 +9,6 @@
 #ifdef WINorDLL
 #if 0
 #define STRICT
-#include <windows.h>
```

Comment:
I think this windows.h should be retained.  It is #if'd out.  Leaving it retains how the scheme is supposed to work.

Reply:
@chipbarnaby this was my suggestion, but I think you're right. There are uses of what looks like Windows functions inside the WINorDLL conditions. Let's add a separate issue to explore how/if to support WINorDLL in the future.